guides/ci/travis.md

```diff
@@ -0,0 +1,238 @@
+
+### Using `latest` and `nightly`
+
+When [creating a Crystal project](../../using_the_compiler/#creating-a-crystal-project) using `crystal init`, Crystal creates a `.travis.yml` file for us:
+
+```yaml
+# .travis.yml
+language: crystal
+
+crystal:
+  - latest
+  - nightly
+
+script:
+  - crystal spec
+  - crystal tool format --check
+```
+
+Although a basic configuration file, it will let us use Travis CI. Now, let's go to Travis CI dashboard to [add the GitHub repository](https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci). From this moment, Travis CI will run the `specs` against the Crystal compiler (using both `latest` and `nightly` releases).
+
+### Using a specific Crystal release
+
+So, let's suppose we want to use version [0.31.1](https://github.com/crystal-lang/crystal/releases/tag/0.31.1) of the Crystal compiler. For this we are going to use [Docker](https://www.docker.com/). First we need to add Docker as a service in `.travis.yml`, and then we can use `docker` commands in our build steps, like this:
+
+```yml
+# .travis.yml
+language: bash
+
+services:
+  - docker
+
+script:
+  - docker run -v $PWD:/src -w /src crystallang/crystal:0.31.1 crystal spec
+```
+
+**Note:** A list with the different official [Crystal docker images](https://hub.docker.com/r/crystallang/crystal/tags) is available at [DockerHub](https://hub.docker.com/r/crystallang/crystal).
+
+### Using `latest`, `nightly` and a specific Crystal release all together!
+
+For using 3 versions of the compiler we are going to use a different configuration file based on [Build Matrix](https://docs.travis-ci.com/user/customizing-the-build#build-matrix)
+Here is the example:
+
+```yaml
+# .travis.yml
+matrix:
+ include:
+   - language: crystal
+     crystal:
+       - latest
+     script:
+       - crystal spec
+
+   - language: crystal
+     crystal:
+       - nightly
+     script:
+       - crystal spec
+
+   - language: bash
+     services:
+       - docker
+     script:
+       - docker run -v $PWD:/src -w /src crystallang/crystal:0.31.1 crystal spec
+```
+
+## Using binary dependencies
+
+Before presenting some examples, it is important to mention that there are many ways to achieve this and **it will heavily depend on our development workflow**.
+
+With this in mind, let's continue!
+
+Using shards dependencies (i.e. libraries declared in `shard.yml`) will not be a problem because, thanks to the `language: crystal` support, Travis CI automatically runs `shards install`.
+
+But, how do we install dependencies outside `shards.yml`?
+Well, here is a first example installing SQLite3 using Travis CI configuration file:
+
+```yaml
+# .travis.yml
+language: crystal
+crystal:
+  - latest
+
+before_install:
+  - sudo apt-get -y install sqlite3
+
+addons:
+  apt:
+    update: true
+
+script:
+  - sqlite3 --version
+  - crystal spec
+```
+
+**Note:** We are using `sqlite3` just for printing SQLite's version, as an example of external dependency.
```

Comment:
```suggestion
**Note:** We are using `apt show` just for printing SQLite's version, as an example of binary dependency.
```

Reply:
I think the idea was that `sqlite3` is installed as an executable. Of course we don't necessarily need that., using `libsqlite` is more precise.
But the wording needs to be changed: using `apt show` to display the installed package is not exactly an example of a binary dependency.

Reply:
Changing "using" with "installing"